mlos_bench/README.md

```diff
@@ -47,152 +54,118 @@ With a [JSON5](https://spec.json5.org) [config file](./mlos_bench/config/) and c
 To get started, we can adapt an example configuration to test out running `mlos-bench`.
 For these instructions, we will be using Azure for our resources.
 
-1. Make sure that you have Azure CLI tool installed and working.
+### 1. Install and activate the conda environment
 
-    > Installation instructions for `az` (Azure CLI) [can be found here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
-
-    If necessary, login to Azure and set your default subscription:
-
-    ```powershell
-    # If using az cli for the first time, a login will be required:
-    az login
-    # Make sure that GSL subscription is your default:
-    az account set --subscription "..."
-    ```
+From here onwards we assume we are in the project root directory.
+Ensure you have a conda environment (`mlos`) set up for executing `mlos_bench`.
+Create and activate the environment with:
 
-2. Generate access tokens to interact with Azure resources.
+```sh
+conda env create -f conda-envs/mlos.yml
+conda activate mlos
+```
 
-    A script at `./scripts/generate-azure-credentials-config.ps1` produces a JSON with Azure credentials.
-    This data is in the format that can be used by our framework.
+### 2. Make sure that you have Azure CLI tool installed and working
 
-    ```powershell
-    # If using for the az cli first time, a login will be required
-    ./scripts/generate-azure-credentials-config.ps1 > ./global_config.json
-    ```
+> Installation instructions for `az` (Azure CLI) [can be found here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
 
-    On Linux, use `./scripts/generate-azure-credentials-config.sh` (requires `az` and `jq` to be installed).
+If necessary, login to Azure and set your default subscription:
 
-3. Copy the sample config tree from `mlos_bench/config` to another folder of your choice, e.g.,
+```powershell
+# If using az cli for the first time, a login will be required:
+az login
+# Make sure that GSL subscription is your default:
+az account set --subscription "..."
+```
 
-    ```sh
-    cp -R mlos_bench/config ./my-config
-    ```
+### 3. Generate access tokens to interact with Azure resources
 
-    > Note: there are several other examples in [`mlos_bench/config`](./mlos_bench/config/) that you can use as a starting point for your own experiments.
+A script at `./scripts/generate-azure-credentials-config.ps1` produces a JSON with Azure credentials.
```

Comment:
Future work: I can probably consolidate those script files so a single `scripts/generate-azure-credentials-config` without an extension will work across both platforms.  We already do this for the devcontainer stuff.

Reply:
I did that here.